.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Do not connect prettier, use only the rules set by the linter.

Reply:
agree, please do it